core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -215,6 +215,10 @@ final case class EitherT[F[_], A, B](run: F[A \/ B]) {
 }
 
 object EitherT extends EitherTInstances {
+  def F[F[_], A, B](a: F[A \/ B]): EitherT[F, A, B] = apply(a)
```

Comment:
This one seems superfluous. There's already `apply` and `eitherT`.

Reply:
it's about consistency... arguably `eitherT` is superfluous because you can just use `apply`.

Reply:
Right. But `eitherT` already exists while `F` does not.